sync.py

```diff
@@ -0,0 +1,95 @@
+import json
+import requests
+import yaml
+from http.server import BaseHTTPRequestHandler, HTTPServer
+
+
+LOKI_GATEWAY_URL = 'http://loki-gateway.glueops-core-loki.svc.cluster.local'
+LOKI_GATEWAY_PORT = '80'
```

Comment:
Can probably drop this since http is automatically port 80 and https is by default port 443.